.github/workflows/build.yml

```diff
@@ -2,12 +2,29 @@ name: GitHub CI
 on: push
 
 jobs:
-  build-windows:
-    name: Build (Windows)
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
```

Comment:
Why is this false? Won't the publish step run if any one of these fail?

Reply:
I think it works differently....
The idea is to let all of the other configs attempt to finish their runs so that if there are additional/repeat errors on other OS it will surface in same build instead of going on a iterative build dance.

But I don't mind removing it if you don't like it.

Reply:
Removed, It is really meaningless for a relatively small project...